python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -36,11 +36,12 @@
     from pyspark.sql.connect.functions import lit
 from pyspark.sql.dataframe import DataFrame
 from pyspark.testing.connectutils import should_test_connect, connect_requirement_message
+from pyspark.testing.pandasutils import PandasOnSparkTestCase
```

Comment:
Hi, @zhengruifeng . Unfortunately, this seems to introduce a pandas dependency accidentally.
Previously, this file is guarded by `if have_pandas` at line 33.